README.md

```diff
@@ -1,10 +1,65 @@
-# photonranch-jobs
+# Photon Ranch Jobs
 
 This repository manages job requests between the web UI and the observatory site software.
 
+## Description
+
+Users can create new job requests, such as pointing a mount or exposing a camera, 
+which observatories can then carry out.
+
+Observatories can retrieve new jobs to perform, update a job's status as it changes, and retrieve recent jobs regardless of status. Use of these API endpoints is mostly found in 
+[the observatory repository](https://github.com/LCOGT/ptr-observatory).
+
+## Architecture
+
+An architecture diagram, when created, will go here.
+
+## Dependencies
+
+Dependencies will be listed here.
+
+## Local Development
+
+Clone the repository to your local machine:
+
+```
+git clone https://github.com/LCOGT/photonranch-jobs.git
+cd photonranch-jobs
+```
+
+### Requirements
+
+You will need the [Serverless Framework](https://www.serverless.com/framework/docs/getting-started) 
+installed locally for development. For manual deployment to AWS as well as for updating dependencies, 
+you will need to install [Node](https://nodejs.org/en/), 
+[npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm), 
+and [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html), 
+configuring with your own AWS credentials.
+
+### Deployment
+
+This project currently has two stages, `dev` (currently treated as the production stage) and `test`. 
+For manual deployment on your local machine, you'll need to fill out the 
+`public_key` and `secrets.json` with the required information, and install packages:
+
+```
+npm install
+serverless plugin install --name serverless-python-requirements
+```
+
+To deploy, run:
+
+```
+serverless deploy --stage {stage}
+```
+
+### Testing
+
+Instructions to manually run tests will be detailed here.
```

Comment:
Add something about how tests are written with pytest, but we only have minimal code coverage. Full tests are planned to be added soon.